.markdownlint.jsonc

```diff
@@ -4,5 +4,6 @@
     "MD004": "dash", // Prefer hyphens for unordered lists
     "MD013": false,  // Exercism tends to break lines at sentence ends rather than at a column limit
     "MD024": false,  // The format for instructions.md requires  repeated headings, e.g. "Task"
-    "MD048": false   // We want three backticks for codeblocks, but four tildes for special blocks.
+    "MD048": false,  // We want three backticks for codeblocks, but four tildes for special blocks.
+    "MD049": false   // Prettier uses underscores, but not all asterisk emphasis can be correctly replaced by underscores.
```

Comment:
I was battling markdownlint in this PR, and then I realized that there's this problem:
```
- The *n*th number gets multiplied by 10^_(n-1)_.
```

Prettier didn't reformat `*n*th` to `_n_th` and when I tried, it doesn't render correctly in my IDE's markdown renderer. So I am making the assumption that I put in the comment here: it is not possible to have consistent emphasis with underscores because apparently they don't work the same as asterisks?

Reply:
Guessing that it is not possible to have prettier enforce asterisks only? 
(Which would result in uniformity)

Reply:
I don't think the emphasis is _that_ important here to be honest

Reply:
It could easily be backticks too I think

Reply:
Prettier offers almost zero configuration options, be design. It is not possible to change its emphasis preference to asterisks.

I'll remove the unnecessary emphasis on _n_ and remove this new linter rule.